src/__stories__/slideshow-story.tsx

```diff
@@ -45,3 +60,52 @@ Default.args = {
     loop: false,
     withBullets: true,
 };
+
+type WithCarouselContextArgs = {numItems: number};
+
+export const WithCarouselContext: StoryComponent<WithCarouselContextArgs> = ({numItems}) => {
+    return (
+        <CarouselContextProvider>
```

Comment:
Maybe we should add some screenshot/acceptance tests to verify that the new functionality works?

Reply:
I tried to cover it with a unit test, but jsdom doesn't support scroll, so it's imposible to cover with unit.
I could write an acceptance test

Reply:
done